site/oauth2-examples.md

```diff
@@ -86,12 +88,14 @@ Run the following 3 commands to get the environment ready to see Oauth 2.0 plugi
 
 The last command starts a RabbitMQ server with [this](https://github.com/rabbitmq/rabbitmq-oauth2-tutorial/blob/main/conf/uaa/rabbitmq.config) configuration file.
 
+## <a id="access-management-ui" class="anchor" href="#access-management-ui">Access Management UI using OAuth 2.0 tokens</a>
 
-## <a id="use-access-tokens" class="anchor" href="#use-oauth-tokens">Use access tokens</a>
+The Management UI can be configured with one of these two login modes:
 
-The following subsections demonstrate how to use access tokens with any messaging protocol and also to access the management ui and rest api.
+* [Service-Provider initiated logon](#service-provider-initiated-logon) - This is the default and traditional OAuth 2.0 logon mode. The user comes to the Management UI and clicks on the button "Click here to logon" which initiates the logon. The logon process starts in RabbitMQ, the *Service Provider*.
```

Comment:
Suggested reword for this sentence: "The user comes to the Management UI and clicks on the button "Click here to logon" which initiates the logon. " CHANGE TO: 
"In the RabbitMQ Management UI, click "Click here to logon" to initiate the logon. "